app/controllers/api/v1/studies_controller.rb

```diff
@@ -85,10 +85,132 @@ def show
+              end
+              property :feature_flags do
+                key :type, :object
+                key :title, 'FeatureFlagOptions'
+                key :description, 'Hash of FeatureFlagOption values with defaults, e.g. flag_name => flag_value'
+              end
+              property :menu_options do
+                key :type, :object
+                key :title, 'MenuOptions'
+                key :description, 'Hash of select menu options for various views'
+                property :fonts do
+                  key :type, :array
+                  items do
+                    key :type, :string
+                  end
+                end
+                property :species do
+                  key :type, :array
+                  items do
+                    key :type, :object
+                    key :title, 'Species information'
+                    property :id do
+                      key :type, :string
+                    end
+                    property :common_name do
+                      key :type, :string
+                    end
+                  end
+                end
+                property :units do
+                  key :type, :array
+                  items do
+                    key :type, :string
+                    key :enum, ExpressionFileInfo::UNITS_VALUES
+                  end
+                end
+                property :library_preparation_protocol do
+                  key :type, :array
+                  items do
+                    key :type, :string
+                    key :enum, ExpressionFileInfo::LIBRARY_PREPARATION_VALUES
+                  end
+                end
+                property :modality do
+                  key :type, :array
+                  items do
+                    key :type, :string
+                    key :enum, ExpressionFileInfo::MODALITY_VALUES
+                  end
+                end
+                property :biosample_input_type do
+                  key :type, :array
+                  items do
+                    key :type, :string
+                    key :enum, ExpressionFileInfo::BIOSAMPLE_INPUT_TYPE_VALUES
+                  end
+                end
+                property :sequence_file_types do
+                  key :type, :array
+                  items do
+                    key :type, :string
+                    key :enum, %w[Fastq BAM]
+                  end
+                end
+                property :genome_assemblies do
+                  key :type, :array
+                  items do
+                    key :type, :object
+                    key :title, 'GenomeAssembly'
+                    property :id do
+                      key :type, :string
+                    end
+                    property :name do
+                      key :type, :string
+                    end
+                    # show ID of associated parent Taxon for storing belongs_to association
+                    property :taxon_id do
```

Comment:
Rather than a PR comment to answer Eric's question, could it be helpful to add a comment or rename the variable to add the type clarity there, e.g.
```suggestion
                    property :taxon_bson_id do
```

Reply:
Good thought - though this attribute is generated automatically via the `has_many / belongs_to` relationship between the `Taxon` and `GenomeAssembly` classes.  We could overwrite it and pass in an attribute name to use for the accession, but that seems a little heavy-handed in this case.